### PR TITLE
Enhance russian translation

### PR DESCRIPTION
**Pull request by nick87720z**
Lathe and Revolve words were maped to Rotate during translation.
However, russian has words to not lose meaning.

### CODE REVIEW

**Comment by ruevs** [comment]
Токарный станок = lathe - и поетому - "Вы&точить". Понятно.
But I'd like to hear what @Evil-Spirit thinks. I have no clue what the proper terminology is in Russian in the context of mechanical engineering.

**Comment by ruevs** [comment]
Relates to #1288

**Comment by nick87720z** [comment]
Yeah, me too.
Somewhat hard case as I don't remember CADs, using such difference
(only tried autocad, kompas and freecad). "Тело точения" or "Тело
поворота" - sound rather unusual despite valid.

**Comment by ruevs** [comment]
@Evil-Spirit do you have an opinion on this?

**Comment by Evil-Spirit** [comment]
"Тело вращения" [solid of revolution](https://www.multitran.com/m.exe?s=solid+of+revolution&l1=1&l2=2) is more common for both lathe operators and students. This is geometrical definition. "Выточить" - this will cause brain damage to every who are not lathe operator. So, need to revert all this since I don't like this at all (pointless to remame to less understandable in order to support initial "lathe" instead of "revolve" )
https://github.com/solvespace/solvespace/pull/1290/commits/7bf427e1f6cece5bb937c2eaece9155fb9bd71cd

**Comment by Evil-Spirit** [comment]
Hey, I just realized that now there is 3 different operations like:

*   revolve by 360
*   revolve by less than 360
*   revolve by more than 360 and shifing by axis

This all should be just single operation! I believe it is possible to merge them into single group without breaking backward compatibility. but if you want to make translation:

Lathe = Тело вращения

Angular lathe / Lathe by angle = Угловое тело вращения  / Тело вращения на угол

File: res/locales/ru_RU.po [hunk 1]
```diff
@@ -932,11 +932,11 @@ msgstr "Тело Винтовое"
 
 #: graphicswin.cpp:118
 msgid "&Lathe"
-msgstr "Тело В&ращения"
```
Comment (Evil-Spirit) [inline res/locales/ru_RU.po:935]: leave this or make it "Тело вращения на 360"

File: res/locales/ru_RU.po [hunk 2]
```diff
@@ -932,11 +932,11 @@ msgstr "Тело Винтовое"
...
+msgstr "Вы&точить"
 
 #: graphicswin.cpp:119
 msgid "Re&volve"
-msgstr "Тело В&ращения"
```
Comment (Evil-Spirit) [inline res/locales/ru_RU.po:939]: Тело вращения на угол / Угловое тело вращения

**Review by Evil-Spirit — CHANGES_REQUESTED** [review]

**Comment by ruevs** [comment]
@nick87720z @Evil-Spirit merged here 31ac8083ae27733b18a6c2f6454104b9efd9fc29 with the discussed changes. I'm pretty sure I did not screw up anything but please take a look.

**Comment by nick87720z** [comment]
@Evil-Spirit Just for interest - how about "Lathe" -> "Токарный станок", that was my original idea. Also most verbatim translation, carrying original author idea totally preserved.

**Comment by ruevs** [comment]
> how about "Lathe" -> "Токарный станок",

But in the English version " lathe" is used as a verb - the literal translation is indeed [выточить]()

**Pull request closed without merging**